README.md

```diff
@@ -0,0 +1,31 @@
+# Code Examples Repository
+
+Welcome to the Venly Code Examples Repository! This repository contains a collection of code examples showcasing various concepts and techniques. Each example is organized into its own folder for easy navigation.
+
+## Getting Started
+
+To run the examples, follow these simple steps:
+
+1. Navigate to the `<example>` folder of your choice.
+
+2. Run the following command to install the necessary dependencies:
+
+    ```bash
+    npm install
+    ```
+
+3. Read the `README.md` file in the specific folder for detailed instructions and information about the example.
+
+4. Once you've followed the specific instructions, run the example using:
+
+    ```bash
+    npm run start
+    ```
+
```

Comment:
this could be different for each example so I would pull it down to the concrete example and not in the overview

Reply:
I would keep it the same for all examples but i will remove it enyway